mne/time_frequency/tfr.py

```diff
@@ -979,16 +980,22 @@ def __init__(self, info, data, times, freqs, nave, comment=None,
         self.preload = True
 
     @verbose
-    def plot(self, picks, baseline=None, mode='mean', tmin=None, tmax=None,
+    def plot(self, picks=None, exclude=None, baseline=None, mode='mean', tmin=None,
+             tmax=None,
              fmin=None, fmax=None, vmin=None, vmax=None, cmap='RdBu_r',
              dB=False, colorbar=True, show=True, title=None, axes=None,
-             layout=None, yscale='auto', verbose=None):
+             layout=None, yscale='auto', verbose=None,
+             aggregate='mean'):
         """Plot TFRs as a two-dimensional image(s).
 
         Parameters
         ----------
-        picks : array-like of int
-            The indices of the channels to plot, one figure per channel.
+        picks : None | array-like of int
+            The indices of the channels to plot, one figure per channel. If
+            None, plot the across-channel average.
+        exclude : None | list of str | 'bads'
```

Comment:
to avoid breaking argument order, `exclude` should probably go just before `verbose` at the end